### PR TITLE
Omer358

### PR DESCRIPTION
**Pull request by omer358**
I noticed a problem with the symbol card, its position is fixed, which causes overlay in RTL language "Arabic for example" with the label of the field, so made it relies on the start & and instead of left & right to support both LTR & RTL

### CODE REVIEW

File: lib/utils/utils_widgets.dart [hunk 1]
```diff
@@ -120,15 +128,16 @@ class UnitCard extends StatelessWidget {
...
+                  const EdgeInsets.only(left: 10.0, right: 10.0, bottom: 10.0),
               child: textField,
             ),
             elevation: 4.0,
           ),
         ),
         if (symbol != null)
           Align(
-            alignment: const Alignment(0.95, -0.9),
+            alignment: AlignmentDirectional.topEnd,
```
Comment (ferraridamiano) [inline lib/utils/utils_widgets.dart:140]:
```suggestion
            alignment: const AlignmentDirectional(0.95, -0.9),
```

Comment (ferraridamiano) [reply lib/utils/utils_widgets.dart:140]: I would not change the visual appearance, just adapt to RTL languages

Comment (omer358) [reply lib/utils/utils_widgets.dart:140]: So this does the same job, but only respect the text direction??

**Comment by ferraridamiano** [comment]
Hi @omer358 and thank you for reporting this.
Some notes, this PR is a one line change but it affects more than 300 lines of code. Please, a 1 line change should concern 1 line. I know that part of the changes is because of the code format extension (I will fix this). Also the name of the PR should be more appropriate.

Anyway, could you please submit a PR with 1 line change?

Thank you!

File: lib/utils/utils_widgets.dart (hunk 1 above)
Comment (ferraridamiano) [reply lib/utils/utils_widgets.dart:140]: Exactly

**Comment by omer358** [comment]
**Sorry about that!** I noticed after I made the pull request.
If you delete this one I'll be happy to make the required changes appropriately👍

**Comment by ferraridamiano** [comment]
No problem, it's ok!
I will close this for now

**Pull request closed without merging**